pkgs/development/python-modules/jax/default.nix

```diff
@@ -29,31 +30,23 @@ buildPythonPackage rec {
     owner = "google";
     repo = pname;
     rev = "jax-v${version}";
-    hash = "sha256-eGdAEZFHadNTHgciP4KMYHdwksz9g6un0Ar+A/KV5TE=";
+    hash = "sha256-pIl7zzl82w5HHnJadH2vtCT4mYFd5YmM9iHC2GoJD6s=";
   };
 
-  patches = [
-    # See https://github.com/google/jax/issues/7944
-    ./cache-fix.patch
-
-    # See https://github.com/google/jax/issues/10292
-    (fetchpatch {
-      url = "https://github.com/google/jax/commit/cadc8046d56e0c1433cf48a2f106947d5f4ecbfd.patch";
-      hash = "sha256-jrpIqt4LzWAswt/Cpwtfa5d1Yn31HcXkVH3ETmaigA0=";
-    })
-  ];
-
   # jaxlib is _not_ included in propagatedBuildInputs because there are
   # different versions of jaxlib depending on the desired target hardware. The
   # JAX project ships separate wheels for CPU, GPU, and TPU. Currently only the
   # CPU wheel is packaged.
   propagatedBuildInputs = [
     absl-py
+    (etils.overrideAttrs (_: {
+      doInstallCheck = false; # needed to break dependency cycle
+    }))
```

Comment:
Please use this https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pylama/default.nix#L70-L72 instead.

Reply:
Actually, both seem unnecessary as there is no dependency cycle (since the etils tests are disabled). This is removed now; we can address the dependency cycle when enabling the etils tests (see https://github.com/NixOS/nixpkgs/pull/183051#issuecomment-1224954563)